hack/istio/functions.sh

```diff
@@ -8,6 +8,7 @@
 # 1. Create a SMM (this is skipped for the cluster wide mode)
 # 2. Label the member namespace (only for the cluster wide mode)
 # 3. Annotate all of the namespace's Deployments with the sidecar injection annotation if enabled
+#! Be sure, that ISTIO_NAMESPACE and (ENABLE_INJECTION or AUTO_INJECTION) are properly set in the script where the function is called!
```

Comment:
This comment seems relevant even for non-Maistra environments (injections is still "a thing" in upstream Istio).

Reply:
the function here is named `prepare_maistra` , so I expected that it is related only to `maistra`. (This function is not in the main branch anymore )